### ISSUE TITLE
Not possible to run this with Node.js prerelease/nightly binaries

### ISSUE THREAD

**Issue by ofrobots** [GoogleCloudPlatform/cloud-profiler-nodejs#221]
While investigating #199, I ran into this issue:

```sh
$ ~/src/node/node --require . testing/busybench.js 600
ERROR:@google-cloud/profiler: Error: Could not start profiler: node version v11.0.0-pre does not satisfies "6.12.3 - 7.0.0 || >=8.9.4"
```

That semver test prevents the profiler from running against Node.js nightly or pre-release build. IMO, this should be a warning rather than a fatal error.

**Comment by aalexand** [comment]
I wonder why "v11.0.0-pre" does not satisfy the ">=8.9.4" part of the clause.

**Comment by ofrobots** [comment]
https://docs.npmjs.com/misc/semver#prerelease-tags

**Comment by nolanmar511** [comment]
Closing.
This should be fixed by #223.

**Comment by aalexand** [comment]
Issues should be closed only after the fixing PR is submitted and #223 is not submitted yet. Re-opening.
